elektronLogging/createLogDir.go

```diff
@@ -0,0 +1,34 @@
+package elektronLogging
+
+import (
+	logrus "github.com/sirupsen/logrus"
```

Comment:
Keep the alias for logrus import consistent. So, refactor this to "log".

Reply:
Done